__tests__/app.test.js

```diff
@@ -17,6 +29,14 @@ describe("api/topics", () => {
       .expect(200)
       .then(({ body }) => {
         const { topics } = body;
+        expect(topics).toHaveLength(3);
+        expect(topics).toBeInstanceOf(Array);
+        topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
+        });
```

Comment:
You should only be making changes relevant to the ticket you are working on. Any changes for previous tickets should be made on those previous branches or on new branches.